.gitignore

```diff
@@ -1,99 +0,0 @@
-# Project ignores
```

Comment:
Typing at the same time @mattgawarecki! I'll add it back and see if there are any file types to be added.